packages/expo-blur/src/BlurView.types.ts

```diff
@@ -17,14 +33,25 @@ export type BlurViewProps = {
   /**
    * A number by which the blur intensity will be divided on Android.
    *
-   * Due to platform differences blurs on Android and iOS vary slightly and might look different
-   * at different intensity levels. This property can be used to fine tune blur intensity on Android to match it
+   * When using experimental blur methods on Android the perceived blur intensity might be different from iOS
+   * at different intensity levels. This property can be used to fine tune it on Android to match it
    * more closely with iOS.
    * @default 4
    * @platform android
    *
    */
   blurReductionFactor?: number;
+
+  /**
+   * Blur method to use on Android.
+   *
+   * Currently, `BlurView` support is experimental on Android and may cause performance and graphical issues.
+   * It can be enabled by setting this property.
```

Comment:
```suggestion
   * > **warning** Currently, `BlurView` support is experimental on Android and may cause performance and graphical issues.
   * Blur method to use on Android.
   *
   * It can be enabled by setting this property.
```

Let's add a warning callout here to be more visible.